deploy/csi-operator.yaml

```diff
@@ -676,7 +676,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v2.0.1
+          image: quay.io/k8scsi/csi-snapshotter:v2.1.1
```

Comment:
do we need this for google analytice?

Reply:
NO .. i was testing this changes and forget to remove, and got added as part of operator changes, will remove them

Reply:
removed